res/css/structures/_MatrixChat.scss

```diff
@@ -76,14 +76,28 @@ limitations under the License.
     height: 100%;
 }
 
+// We'd like to remove this, but this makes matrixchat's resizehandle's
+// negative margin & greater than its positive padding. If it's the same
+// or less, Safari gets confused about overflows somehow and
+// https://github.com/vector-im/element-web/issues/19863 happens.
+.mx_MatrixChat > .mx_ResizeHandle.mx_ResizeHandle_horizontal {
+    margin: 0 calc(-6px - $container-gap-width / 2) 0 calc(-6px + $container-gap-width / 2);
+    // This is required to make WebKit not bleed the blurred bg of the leftPanel!
+    // The condition to prevent bleeding is: (margin-left+maring-right < 11px) (IF there is NO margin on the leftPanel_wrapper)
```

Comment:
maring -> margin

I am a bit puzzled parsing this, aren't the margin values negative...?

Reply:
they are It should be -11px. Thanks!